src/components/Comments/WriteComment.js

```diff
@@ -84,13 +85,11 @@ const WriteComment = ({
           {username}
         </Text>
         <View style={styles.anonimityContainer}>
-          <ToggleSwitch
+          <ToggleSwitchAnon
             value={isAnonimity}
             onValueChange={toggleSwitch}
             labelLeft="Anonymity"
-            backgroundActive={colors.lightgrey}
-            backgroundInactive={colors.lightgrey}
-            styleLabelLeft={styles.switch}
+            labelColor={COLORS.gray8}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `ToggleSwitchAnon` component has been used instead of the old `ToggleSwitch` component. The `labelColor` prop has been added, which was not present in the old code. Ensure that this new prop is handled correctly in the `ToggleSwitchAnon` component.
```diff
- <ToggleSwitch
+ <ToggleSwitchAnon
    value={isAnonimity}
    onValueChange={toggleSwitch}
    labelLeft="Anonymity"
-   backgroundActive={colors.lightgrey}
-   backgroundInactive={colors.lightgrey}
-   styleLabelLeft={styles.switch}
+   labelColor={COLORS.gray8}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->